frontend/plugins/operation_ui/src/modules/navigation/TeamsNavigation.tsx

```
@@ -144,6 +146,13 @@ const TeamActionsMenu = ({ team }: { team: Team }) => {
     }
   };
 
+  const handleRemoveMember = () => {
+    const currentUser = useAtomValue(currentUserState);
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)